admin/test/test_acceptance.py

```diff
@@ -205,3 +207,19 @@ def test_non_json_message(self):
             )],
             self._convert(NON_JSON_JOURNAL_EXPORT),
         )
+
+
```

Comment:
We didn't have any tests for CommonOptions that I could find, but I decided it would be a good idea to add one anyway